compiler-wasm/Cargo.toml

```diff
@@ -22,6 +22,7 @@ itertools.workspace = true
 serde.workspace = true
 termcolor.workspace = true
 tracing.workspace = true
+getrandom.workspace = true
```

Comment:
How come this has been added? This package is unmodified and doesn't use random numbers afaik

Reply:
This is from the required update to the latest version of hexpm-rust. After the dependencies in hexpm-rust were bumped there was a new indirect dependency added `getrandom` which required the `js` flag to be added in order for builds on the `wasm32-unknown-unknown` target to work. I added comments to the root Cargo.toml to explain this:
https://github.com/gleam-lang/gleam/blob/218ebb06827c983631dea8ebf6c0ef1abcd3b8a1/Cargo.toml#L61-L62

Reply:
Also this diff is inaccurate now because the PR that updates the CI got merged which also required this same hexpm-rust bump. So this PR now has no additional effect on this stuff. I have rebased this PR to give an updated diff